src/app/header/header.component.html

```diff
@@ -47,7 +48,7 @@ <h5 id="offcanvasStartLabel">penda</h5>
     <br />
     <div class="offcanvas-body">
         <div class="list-group list-group-flush">
-            <div class="dropdown">
```

Comment:
I assume the `dropdown` class was removed to get rid of the box shadow? If so then the styling for it should also be removed, i.e. the following code should be then deleted as I think it is essentially dead code?
```
.dropdown div {
    box-shadow: 0 0 1px 1px #e3e3e3ae;
}
```
Additionally, see the below comment for writing `class=""`.